### ISSUE TITLE
Document external KiCAD symbols

### ISSUE THREAD

**Issue by ericmutton** [k5utd/Cobalt-Cowboy#4]
We should build a documentation section for where the symbols and libraries are maintained if external to KiCAD sources.

https://github.com/espressif/kicad-libraries

_Originally posted by @ericmutton in https://github.com/k5utd/Cobalt-Cowboy/pull/3#pullrequestreview-1663847885_

**Comment by AbleArcher88** [comment]
I agree. While espreesif's libraries are a good place to find them, we should try to compile a single cobalt cowboy library to ensure any one who opens the project can see them.
